javalin/src/test/java/io/javalin/TestRouting.kt

```diff
@@ -214,12 +221,12 @@ class TestRouting {
     @Test
     fun `proposal works as expected`() = TestUtil.test { app, http ->
         app.get("/a-*") { ctx -> ctx.result("A") }
-        app.get("/b-**") { ctx -> ctx.result("B") }
+        app.get("/b-%") { ctx -> ctx.result("B") }
         app.get("/c-<param>") { ctx -> ctx.result("C" + ctx.pathParam("param")) }
         app.get("/d-{param}") { ctx -> ctx.result("D" + ctx.pathParam("param")) }
         app.get("/e-<param>-end") { ctx -> ctx.result("E" + ctx.pathParam("param")) }
         app.get("/f-{param}-end") { ctx -> ctx.result("F" + ctx.pathParam("param")) }
-        app.get("/g-***") { ctx -> ctx.result("G") }
+        app.get("/g-*") { ctx -> ctx.result("G") }
```

Comment:
This should be `/g-*%` to keep the old logic.

Reply:
What is that line testing? The fact that the test still passes suggested "not much" 😅